src/agent/scm/svn.ts

```diff
@@ -0,0 +1,277 @@
+import Q = require('q');
+import scmprovider = require('./lib/scmprovider');
+import agentifm = require('vso-node-api/interfaces/TaskAgentInterfaces');
+import sw = require('./lib/svnwrapper');
+import cm = require('../common');
+import utilm = require('../utilities');
+
+var shell = require('shelljs');
+var path = require('path');
+var tl = require('vso-task-lib');
+
+var administrativeDirectoryName = ".svn";
```

Comment:
This is also declared in svnwrapper.  Maybe export it from there so it's only declared once?